Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:8.1
+# Usage:
+# Use existing host SSL keys
+# docker run -d -p 8081:8081 -v /etc/letsencrypt/:/etc/letsencrypt/ --name "nimiq" nimiq
+# Mount volume or data until dynamic private key can be used
+# docker run -d -p 8080:8080 -v /etc/letsencrypt/:/etc/letsencrypt/ -v /data/:/core/nimiq/core/dist/database --name "nimiq" nimiq
+
+
+ENV GIT_REPO="https://github.com/nimiq-network/core"
+ENV DOMAIN="node.nimiq.io"
+ENV KEY="/etc/letsencrypt/live/nimiq.io/privkey.pem"
+ENV CRT="/etc/letsencrypt/live/nimiq.io/cert.pem"
+ENV PORT="8080"
+
+RUN apt-get update && apt-get -y upgrade
+RUN apt-get install -y git python openssl build-essential software-properties-common
```

Comment:
You can completely skip the need for `apt-get update`, `upgrade`, and `install` just by downloading the master zip. 

I will always be under `https://github.com/nimiq-network/core/archive/master.zip`.